i18n/locales/en.json

```diff
@@ -116,6 +116,9 @@
   "mostRecentAlertsShownIn": "Most recent alerts shown on map in",
   "notes": "Notes",
   "noColumnEntry": "No column entry",
+  "noDescriptionProvidedYet": "No description provided yet.",
+  "addDescription": "Add description",
+  "contactAdminToAddDescription": "Please contact an admin to add a description.",
```

Comment:
As has been mentioned before: please always add add these new translations in alphabetical order.